includes/compat/class-wc-stripe-subs-compat.php

```diff
@@ -257,10 +261,31 @@ public function process_subscription_payment( $amount = 0.0, $renewal_order, $re
 				throw new WC_Stripe_Exception( print_r( $response, true ), $localized_message );
 			}
 
-			do_action( 'wc_gateway_stripe_process_payment', $response, $renewal_order );
+			// Either the charge was successfully captured, or it requires further authentication.
+
+			if ( $is_authentication_required ) {
+				do_action( 'wc_gateway_stripe_process_payment_authentication_required', $renewal_order, $response );
 
+				$error_message = 'This transaction requires authentication.';
```

Comment:
This message should be internationalized.

Reply:
Fixed.